src/cast_core/include/cast_core/actors/CollectionScanner.hpp

```diff
@@ -50,16 +50,17 @@ class CollectionScanner : public Actor {
     static std::string_view defaultName() {
         return "CollectionScanner";
     }
+    struct PhaseConfig;
```

Comment:
I doubt this needs to be public?

Reply:
it does otherwise you can't access it in the genny::actor scope.